### ISSUE TITLE
Dataset cited paper in acl/pdf_parses.jsonl 

### ISSUE THREAD

**Issue by Richard-LZ-Zhang** [jacklxc/CORWA#2]
I followed your instruction to extract the pdf_parses for papers in ACL, from the S2ORC dataset, obtaining a "20200705v1/acl/pdf_parses.jsonl". However, I notice that not all cited and citing papers in your dataset (e.g., test dataset CORWA_test.jsonl) can be found in this file (i.e., the ids listed in the CORWA_test.jsonl for both cited and citing papers are not found in acl/pdf_parses.jsonl). In the LED generation process, you need to take as input the abstract / introduction of both cited and citing paper, and if I understand correctly, you at least need to extract the abstract and introduction of all the citing and cited papers in the test dataset, from any pdf_parses file. Should I just scan through the entire "20200705v1/full/pdf_parses/* " to obtain such information?  Thank you very much in advance!

**Comment by jacklxc** [comment]
`20200705v1/acl/pdf_parses.jsonl` is simply the filtered version of `20200705v1/full/pdf_parses/*` by only selecting those papers with ACL IDs. CORWA-train and -test are human-annotated, and they are selected by taking those related work sections that have the fewest number of missing cited papers that are not covered by S2ORC. All the rest of the S2ORC ACL papers are in the distant set. The training and test set is split by year (before & after 2019).

Originally I hesitated to share `related_work.jsonl` files because they are a simple derivation of S2ORC's `20200705v1/full/pdf_parses/*`, but eventually all this information is included in the latest released `CORWA_{train, test, distant}.jsonl` files.

**Comment by jacklxc** [comment]
Note that many wired file formats exist because of historical reasons during my project development. Unless you are interested in replicating my exact experiment, you can start from `data/CORWA_{train, test, distant}.jsonl` for your custom usage. You can use the code under `pipeline/` to tag your own related work sections with CORWA annotations.

**Comment by Richard-LZ-Zhang** [comment]
Thank you for your reply. I am also interested in testing my model on your test dataset, and making comparison with your finetuned LED. Would you please provide more information on how the ROUGE score of LED generation reported in your paper was obtained? For each span in the test dataset, if one or more references are not in S2ORC or ACL of S2OCR, did you just leave them out?

**Comment by jacklxc** [comment]
@mandalbiswadip Can you resolve this request? Thanks.

**Comment by mandalbiswadip** [comment]
I have updated the evalution script under `experiments/evaluate_LED.ipynb`.  Let me know if you find this helpful

**Comment by Richard-LZ-Zhang** [comment]
Horray! That looks great!
